### PR TITLE
dashboard

### PR DESCRIPTION
**Pull request by zippiL**

### CODE REVIEW

File: src/components/marketing/layout/layout.tsx [hunk 1]
```diff
@@ -24,9 +24,9 @@ export function Layout({ children }: LayoutProps): React.JSX.Element {
         }}
       />
       <div>
-        <MainNav />
+        {/* <MainNav /> */}
```
Comment (benshabatnoam) [inline src/components/marketing/layout/layout.tsx:27]: Don't leave comment out code

Comment (benshabatnoam) [reply src/components/marketing/layout/layout.tsx:27]: This component is not in use elsewhere you can remove it from code

Comment (benshabatnoam) [reply src/components/marketing/layout/layout.tsx:27]: Remove the import of it in line 5

File: src/components/marketing/layout/layout.tsx [hunk 2]
```diff
@@ -24,9 +24,9 @@ export function Layout({ children }: LayoutProps): React.JSX.Element {
...
         {children}
-        <Footer />
+        {/* <Footer /> */}
```
Comment (benshabatnoam) [inline src/components/marketing/layout/layout.tsx:29]: Same for this one

**Review by benshabatnoam — CHANGES_REQUESTED** [review]

File: src/components/marketing/home/listMinyan.tsx [hunk 3]
```diff
@@ -0,0 +1,140 @@
...
+
+interface Order {
+    id: string;
+    customer: { name: string; email: string };
+    paymentMethod: string;
+    currency: string;
+    totalAmount: number;
+    status: 'pending' | 'completed' | 'canceled';
+    createdAt: Date;
+}
```
Comment (benshabatnoam) [inline src/components/marketing/home/listMinyan.tsx:28]: Replcae Order interface with Minyan interface

File: src/components/marketing/home/room.tsx [hunk 4]
```diff
@@ -0,0 +1,114 @@
...
+        id: 'ASSET-001',
+        mimeType: 'image/png',
+        name: 'example-project2.png',
+        size: '3.2 MB',
+        url: '/assets/image-minimal-2.png',
+        status: 'off'
+    },
+];
+
+export function Room(): React.JSX.Element {
```
Comment (benshabatnoam) [inline src/components/marketing/home/room.tsx:50]: Change to RoomMatrix it's more desriptive

Comment (benshabatnoam) [reply src/components/marketing/home/room.tsx:50]: Make sure to change the file name as well

File: src/components/marketing/layout/main-nav.tsx [hunk 5]
```diff
@@ -1,193 +0,0 @@
...
-import { isNavItemActive } from '@/lib/is-nav-item-active';
-import { usePathname } from '@/hooks/use-pathname';
-import { Dropdown } from '@/components/core/dropdown/dropdown';
-import { DropdownPopover } from '@/components/core/dropdown/dropdown-popover';
-import { DropdownTrigger } from '@/components/core/dropdown/dropdown-trigger';
-import { RouterLink } from '@/components/core/link';
-import { Logo } from '@/components/core/logo';
-
-import { MobileNav } from './mobile-nav';
-import { PagesPopover } from './pages-popover';
```
Comment (benshabatnoam) [inline src/components/marketing/layout/main-nav.tsx:24]: This component is not in use elsewhere therefore please remove it.
NOTE: whenever you're removing an unused component check if it contains other imports that aren't in use and delete them as well.

File: src/components/marketing/home/room.tsx [hunk 6]
```diff
@@ -0,0 +1,114 @@
...
+    },
+    {
+        id: 'ASSET-001',
+        mimeType: 'image/png',
+        name: 'example-project2.png',
+        size: '3.2 MB',
+        url: '/assets/image-minimal-2.png',
+        status: 'off'
+    },
+];
```
Comment (benshabatnoam) [inline src/components/marketing/home/room.tsx:48]: Change the interface and its props to RoomAction and its props

File: src/components/marketing/home/room.tsx [hunk 7]
```diff
@@ -0,0 +1,114 @@
...
+        url: '/assets/image-abstract-2.png',
+        status: 'on'
+    },
+    {
+        id: 'ASSET-002',
+        mimeType: 'application/zip',
+        name: 'docs.zip',
+        size: '6.2 MB',
+        url: '#',
+        status: 'blur'
```
Comment (benshabatnoam) [inline src/components/marketing/home/room.tsx:38]: 'status' should be boolean and renamed to 'active'

File: src/pages/marketing/home.tsx [hunk 8]
```diff
@@ -3,13 +3,10 @@ import { Helmet } from 'react-helmet-async';
...
-import { Included } from '@/components/marketing/home/included';
-import { Productivity } from '@/components/marketing/home/productivity';
-import { StartBuilding } from '@/components/marketing/home/start-building';
-import { Testimonails } from '@/components/marketing/home/testimonials';
+
+import { Room } from '@/components/marketing/home/room';
+import { ListMinyan } from '@/components/marketing/home/listMinyan';
+import { Box, Grid } from '@mui/material';
 
 const metadata = { title: config.site.name, description: config.site.description } satisfies Metadata;
```
Comment (benshabatnoam) [inline src/pages/marketing/home.tsx:11]: Rename marketing folder name to home

File: src/config.ts [hunk 9]
```diff
@@ -33,7 +33,7 @@ export interface Config {
 export const config = {
   site: {
     name: 'Devias Kit Pro',
-    description: '',
+    description: 'בנין בראך',
```
Comment (benshabatnoam) [inline src/config.ts:36]: Get this description from env file

File: src/root.tsx [hunk 10]
```diff
@@ -40,7 +40,7 @@ export function Root({ children }: RootProps): React.JSX.Element {
               <I18nProvider language="en">
                 <ThemeProvider>
                   {children}
-                  <SettingsButton />
+                  {/* <SettingsButton /> */}
```
Comment (benshabatnoam) [inline src/root.tsx:43]: Remove comment and component if needed

**Review by benshabatnoam — CHANGES_REQUESTED** [review]

**Review by zippiL — COMMENTED** [review]
corrections

File: src/components/marketing/home/room.tsx (hunk 7 above)
Comment (benshabatnoam) [reply src/components/marketing/home/room.tsx:38]: @zippiL is this done?

Comment (zippiL) [reply src/components/marketing/home/room.tsx:38]: yes

**Pull request closed without merging**